lib/capn-malloc.c

```diff
@@ -25,6 +25,10 @@ struct check_segment_alignment {
 };
 
 static struct capn_segment *create(void *u, uint32_t id, int sz) {
+	// Silence warnings about unused parameters.
```

Comment:
I personally like -Wunused-parameters :) , and so compile my project with it, but I'm happy to use -Wno-unused-parameters here. I presume it would be fine to set it for the file with a an `#ifdef __GNUC__` wrapped #pragma?